linters/node/gnostic-lint-responses/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "gnostic-lint-responses",
+  "version": "1.0.0",
+  "description": "Gnostic linter plugin to check responses",
+  "main": "gnostic-lint-responses.js",
+  "dependencies": {
+    "get-stdin": "^5.0.1",
+    "lodash": "^4.17.5",
+    "nexe": "^2.0.0-rc.24",
+    "protobufjs": "^6.8.4"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
For consistency with the overall project, can we replace "ISC" with "Apache-2.0" here?